src/plugins/intel_gpu/src/graph/include/layout_optimizer.h

```diff
@@ -209,5 +209,8 @@ class layout_optimizer {
     /// As an example, if a reorder has 2 reorder users that each reorder has a convolution user, then merge is not done when those 2 convolutions
     /// have different result from this function.
     bool needs_onednn_bfyx_to_blocked(format fmt_prev, format fmt_next, layout& prev_output_layout, const convolution_node& node);
+    /// @brief Validates all user node of the target 'node'.
+    /// All user nodes are convolutions which satisfy options for onednn first conv using 'needs_onednn_bfyx_to_blocked'.
+    bool needs_all_usr_onednn_bfyx_to_blocked(format fmt_prev, const program_node& node);
```

Comment:
If it is used only within layout_optimizer, what about making it as static function?

Reply:
I perfer this type cuz it is using needs_onednn_bfyx_to_blocked. Does it need to be fixed?